### ISSUE TITLE
bug: libz package not found

### ISSUE THREAD

**Issue by dmigous** [Softmotions/ejdb#151]
I just cross compiled ejdb for iOS from Linux host, and had error on linkage of executables

```
Linking C executable jbtmgr
Undefined symbols for architecture arm64:
  "_crc32", referenced from:
      __tc_getcrc_impl in libejdb-1.a(myconf.c.o)
  "_deflate", referenced from:
      __tc_deflate_impl in libejdb-1.a(myconf.c.o)
     (maybe you meant: __tc_deflate)
  "_deflateEnd", referenced from:
      __tc_deflate_impl in libejdb-1.a(myconf.c.o)
  "_deflateInit2_", referenced from:
      __tc_deflate_impl in libejdb-1.a(myconf.c.o)
  "_inflate", referenced from:
      __tc_inflate_impl in libejdb-1.a(myconf.c.o)
     (maybe you meant: __tc_inflate)
  "_inflateEnd", referenced from:
      __tc_inflate_impl in libejdb-1.a(myconf.c.o)
  "_inflateInit2_", referenced from:
      __tc_inflate_impl in libejdb-1.a(myconf.c.o)
```

But  then I added -lz to LDFLAGS environment variable and it helped. Maybe somwhere in CMakeLists there is missing -lz flag?

I guess it is because there is in CMakeLists.txt use of

```
find_package(ZLIB)
if (ZLIB_FOUND)
 list(APPEND EJDB_LLIBRARIES ${ZLIB_LIBRARIES})
 list(APPEND EJDB_INCLUDE_DIRS ${ZLIB_INCLUDE_DIR})
endif(ZLIB_FOUND)
```

So there is no cmake-package inside iOS SDK, but just libz.dylib. Hence it doesn't finds libz and proceeds build. So from my point of few it should do one of following:
- give an error if not found;
- give a warning and don't build executables;
- use find_library() here.

I would like to send even pull request for it, but don't know your vision upon it.

**Comment by rennhak** [comment]
+1

**Comment by adamansky** [comment]
Closed in favour to ejdb 2.0 https://github.com/Softmotions/ejdb/blob/master/README.md
